### PR TITLE
Follow stan hint, explicit fixity (default is infixl 9).

### PR DESCRIPTION
**Pull request by philderbeast**
See #610. I'm putting this up here as-is so please label with WIP. I'd like some feedback on which fixities to go with. Are there some that shouldn't be associative? Could some of these be replaced with existing operators we pull in from elsewhere?

https://kowainik.github.io/posts/fixity#standard-examples

For now I've just been explicit but with the default `infixl 9`.

### CODE REVIEW

File: src/Language/Fixpoint/Misc.hs [hunk 1]
```diff
@@ -425,12 +425,15 @@ allCombinations xs = assert (all ((length xs == ) . length)) $ go xs
 powerset :: [a] -> [[a]]
 powerset xs = filterM (const [False, True]) xs
 
+infixl 9 =>>
 (=>>) :: Monad m => m b -> (b -> m a) -> m b
 (=>>) m f = m >>= (\x -> f x >> return x)
 
+infixl 9 <<=
 (<<=) :: Monad m => (b -> m a) -> m b -> m b
 (<<=) = flip (=>>)
```
Comment (facundominguez) [inline src/Language/Fixpoint/Misc.hs:423]: I would think these should be removed from here. And replaced in the code with `>>=` and `=<<`.

File: src/Language/Fixpoint/Misc.hs [hunk 2]
```diff
@@ -425,12 +425,15 @@ allCombinations xs = assert (all ((length xs == ) . length)) $ go xs
...
 
+infixl 9 <$$>
 (<$$>) ::  (Monad m) => (a -> m b) -> [a] -> m [b]
 _ <$$> []           = return []
 f <$$> [x1]         = singleton <$> f x1
```
Comment (facundominguez) [inline src/Language/Fixpoint/Misc.hs]: Maybe it is worth replacing this with `traverse` in the code.

**Comment by philderbeast** [comment]
Looks like `(=>>) :: Monad m => m b -> (b -> m a) -> m b` and `(<<=)` are not used.

File: src/Language/Fixpoint/Misc.hs (hunk 1 above)
Comment (philderbeast) [reply src/Language/Fixpoint/Misc.hs:423]: Looks like `(=>>) :: Monad m => m b -> (b -> m a) -> m b` and `(<<=)` are not used.

Comment (philderbeast) [reply src/Language/Fixpoint/Misc.hs:423]: Had a quick peek at `liquidhaskell`. Each operator is only used once there, `(=>>)` in `Language.Haskell.Liquid.Constraint.Fresh` and `(<<=)` in `Language.Haskell.Liquid.Constraint.Generate`. Perhaps those modules should define such an operator privately or use the inline equivalent and we remove them from `liquid-fixpoint`?

**Review by facundominguez — APPROVED** [review]
LGTM!

**Pull request merged**